app/models/miq_server.rb

```diff
@@ -625,7 +625,7 @@ def display_name
   end
 
   def server_timezone
-    get_config("vmdb").config.fetch_path(:server, :timezone) || "UTC"
+    @timezone ||= get_config("vmdb").config.fetch_path(:server, :timezone) || "UTC"
```

Comment:
How would this change for the handful of times when the user actually changes the server's timezone in the settings?  Shouldn't the `@timezone` ivar be reset to nil when the settings get updated?